src/model/control_desk.py

```diff
@@ -284,28 +311,32 @@ def unlink(self) -> bool:
             else:
                 i += 1
         BankSet._fish_connector.send_fader_bank_set_delete_message(self.id)
-        BankSet._linked_bank_sets.pop(found_index)
+        BankSet._linked_bank_sets.pop(found_index)  # TODO found index koente leer sein
         return True
 
     @property
     def is_linked(self) -> bool:
+        """TODO docstring"""
         return self.pushed_to_fish
 
 
 def set_network_manager(network_manager: NetworkManager):
+    """TODO docstring"""
     BankSet._fish_connector = network_manager
 
 
 def set_seven_seg_display_content(content: str):
+    """TODO docstring"""
     BankSet._seven_seg_data = content[0:12] + " " * (12 - len(content))
     # TODO send update message
 
 
-def commit_all_banksets():
+def commit_all_bank_sets():
+    """TODO docstring"""
```

Comment:
"""This method calls update on all linked columns.

This is useful in the event of a reconnect to fish as the state of fish is unknown at this point in time.
"""